src/main/java/nl/knaw/dans/filemigration/cli/LoadFromDataverseCommand.java

```diff
@@ -0,0 +1,113 @@
+import nl.knaw.dans.filemigration.core.DataverseLoaderImpl;
+import nl.knaw.dans.filemigration.db.ActualFileDAO;
+import nl.knaw.dans.lib.dataverse.DataverseClient;
+import nl.knaw.dans.lib.dataverse.SearchOptions;
+import nl.knaw.dans.lib.dataverse.model.search.DatasetResultItem;
+import nl.knaw.dans.lib.dataverse.model.search.ResultItem;
+import nl.knaw.dans.lib.dataverse.model.search.SearchItemType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.Iterator;
+import java.util.Spliterator;
+import java.util.stream.Stream;
+
+import static java.util.Collections.singletonList;
+import static java.util.Spliterators.spliteratorUnknownSize;
+import static java.util.stream.StreamSupport.stream;
+
+public class LoadFromDataverseCommand extends EnvironmentCommand<DdVerifyFileMigrationConfiguration> {
+
+    private static final Logger log = LoggerFactory.getLogger(LoadFromDataverseCommand.class);
+    private final HibernateBundle<DdVerifyFileMigrationConfiguration> verificationBundle;
+
+    /**
+     * Creates a new environment command.
+     *
+     * @param application the application providing this command
+     */
+    public LoadFromDataverseCommand(
+        Application<DdVerifyFileMigrationConfiguration> application,
+        HibernateBundle<DdVerifyFileMigrationConfiguration> verificationBundle
+    ) {
+        super(application, "load-from-dataverse", "Load actual table with file info from dataverse");
+        this.verificationBundle = verificationBundle;
+    }
+
+    @Override
+    public void configure(Subparser subparser) {
+        // mandatory variant of: super.configure(subparser);
+        subparser.addArgument("file")
+            .type(File.class)
+            .required(true)
+            .help("application configuration file");
+
+        subparser.addArgument("-d", "--doi")
+            .dest("doi")
+            .nargs("?")
+            .help("The DOI for which to load the files");
+    }
+
+    @Override
+    protected void run(Environment environment, Namespace namespace, DdVerifyFileMigrationConfiguration configuration) throws Exception {
+        // https://stackoverflow.com/questions/42384671/dropwizard-hibernate-no-session-currently-bound-to-execution-context
+        DataverseClient client = configuration.getDataverse().build();
+        DataverseLoader proxy = new UnitOfWorkAwareProxyFactory(verificationBundle)
+            .create(
+                DataverseLoaderImpl.class,
+                new Class[] { DataverseClient.class, ActualFileDAO.class },
+                new Object[] { client, new ActualFileDAO(verificationBundle.getSessionFactory()) }
+            );
+        String doi = namespace.getString("doi");
+        if (doi != null) proxy.loadFromDataset(doi);
+        else {
+            log.info("No DOI provided, loading all datasets");
+            Iterator<ResultItem> resultItems = client.search().iterator("datasetVersionId:1", datasetOption());
```

Comment:
Dar delivers the (manualy created?) of the base box, demo returns nothing:
https://dar.dans.knaw.nl/api/search?q=datasetVersionId:1&per_page=10&start=0&type=dataset
https://demo.archaeology.datastations.nl/api/search?q=datasetVersionId:1&per_page=10&start=0&type=dataset